### PR TITLE
Add updateCommunityBuild command

### PR DESCRIPTION
**Pull request by OlivierBlanvillain**
This PR adds `community-build/updateCommunityBuild` to address @smarter's comment in https://github.com/lampepfl/dotty-drone/pull/14

### CODE REVIEW

File: project/Build.scala [hunk 1]
```diff
@@ -920,6 +922,13 @@ object Build {
         IO.write(baseDirectory.value / "sbt-dotty-sbt", pluginText)
         IO.write(baseDirectory.value / "dotty-bootstrapped.version", dottyVersion)
       },
+      updateCommunityBuild := testOnly.in(Test).toTask(
+        " " + "dotty.communitybuild.CommunityBuildUpdate" +
+        " " + "-- --inclue-categories=dotty.communitybuild.UpdateCategory").value,
```
Comment (smarter) [inline project/Build.scala]: typo ? inclue -> include (same below)

Comment (smarter) [reply project/Build.scala]: Also I'd just put all of this in one String, makes it easier to copy-paste into sbt

File: community-build/test/scala/dotty/communitybuild/CommunityBuildTest.scala [hunk 2]
```diff
@@ -147,3 +149,12 @@ class CommunityBuildTest {
...
   // )
 }
+
+class TestCategory
+class UpdateCategory
+
+@Category(Array(classOf[UpdateCategory]))
+class CommunityBuildUpdate extends CommunityBuildTest {
+  override def test(project: String, command: String): Unit =
+    super.test(project, command.split("/").head + "/update")
```
Comment (smarter) [inline community-build/test/scala/dotty/communitybuild/CommunityBuildTest.scala]: That's not correct for https://github.com/lampepfl/dotty/blob/38189b5a6b4556a94f998676206d4d59b8b09aa5/community-build/test/scala/dotty/communitybuild/CommunityBuildTest.scala#L89

`jvm/update` won't download the test dependencies, you need `jvm/test:update` to be complete.
I suggest changing the `test` method to be used like this to be more general:
```scala
  @Test def scalatest = test(
    project = "scalatest",
    testCommand = "jvm/test:compile",
    updateCommand = "jvm/test:update"
)
```
It's a bit more verbose but there's no surprise this way.

Comment (OlivierBlanvillain) [reply community-build/test/scala/dotty/communitybuild/CommunityBuildTest.scala]: :+1:

**Pull request merged**